sys-cluster/teleport/Manifest

```diff
@@ -1 +1 @@
-DIST teleport-2.4.2.tar.gz 9975909 BLAKE2B 68916e1d15f49448e918b39b2a760f90eca292b07f61e51cfe125eb99c6b36d95a528cfec40b62cd61893ce81edbf5f26066eb59063214369402e2a62d07bd5e SHA512 f9c2f923e05c8fa0b82708955c944e1bd35da2e6bf8b673832e76c91b918a4bc6d6666797faf0f7572cfaa7341991ef8937cd9b374cdc273271aff45f96b2960
```

Comment:
Please don't do moves when bumping version. One commit that adds each new version, and one commit that cleans all old versions you want to clean. That said, I would recommend keeping the previous newest version for a while in case something went wrong.

Reply:
2.4.2 -> 2.4.5 was only minor bugfixes, though keeping old latest version is a valid point.  Regardless, there was an urgent release of v2.5.2 today to fix a security issue. I'll close this PR and re-submit with the correct way and an updated branch name.

Reply:
New PR: #7481